mmv1/products/metastore/Service.yaml

```diff
@@ -314,10 +317,13 @@ properties:
         description: |
           A Cloud Storage URI of a folder, in the format gs://<bucket_name>/<path_inside_bucket>. A sub-folder <backup_folder> containing backup files will be stored below it.
         required: true
-  - name: 'deletionProtection'
+  - name: 'deletion_protection'
     type: Boolean
     description: |
-      Indicates if the dataproc metastore should be protected against accidental deletions.
+      Indicates if the dataproc metastore should be protected against accidental deletions.Defaults to false.
+      When the field is set to true in Terraform state, a `terraform apply`
+      or `terraform destroy` that would delete the service will fail.
+    default_value: false
```

Comment:
Adding a default value of false would not be a breaking change. Only making it default to true would be breaking.

As far as I know, the default value is already false.